python/nav/models/manage.py

```diff
@@ -1617,9 +1649,17 @@ class SwPortVlan(models.Model):
 
     id = models.AutoField(db_column='swportvlanid', primary_key=True)
     interface = models.ForeignKey(
-        'Interface', on_delete=models.CASCADE, db_column='interfaceid'
+        'Interface',
+        on_delete=models.CASCADE,
+        db_column='interfaceid',
+        related_name="sw_port_vlans",
```

Comment:
See my previous comment about how `gwport` and `swport` without underscores are pretty universal terms in the codebase.